packages/energy-api-influxdb/src/reads/reads.service.ts

```diff
@@ -118,6 +133,20 @@ export class ReadsService implements OnModuleInit {
     `;
   }
 
+  public findLastReadByMeterQuery(
+    meterId: string,
+    startDuration?: AllowedDurationType,
+  ) {
+    const start = startDuration ? `-${startDuration}` : '-1d';
+    return `
+    from(bucket: "${this.bucket}")
+    |> range(start: ${start}, stop: now())
+    |> limit(n: 10)
```

Comment:
What's the purpose for range and limit when using `last` ? Is `last` same as `top(1)` ?

Reply:
Without range the query won`t work. On Flux they say:
_Flux requires a time range when querying time series data. “Unbounded” queries are very resource-intensive and as a protective measure, Flux will not query the database without a specified range._

I think limit can be removed.

I don`t really want to use range, I just want to retrieve the last element in the database for a specific meter Id, and this was the only solution I found. But I am open to other ideas.